core/client/hdfs/src/test/java/alluxio/hadoop/AbstractFileSystemTest.java

```diff
@@ -585,6 +587,46 @@ public void getBlockLocationsNoUfsLocationsWithFallback() throws Exception {
     }
   }
 
+  @Test
+  public void appendExistingNotSupported() throws Exception {
+    Path path = new Path("/file");
+    alluxio.client.file.FileSystem alluxioFs =
+        mock(alluxio.client.file.FileSystem.class);
+    when(alluxioFs.exists(new AlluxioURI(HadoopUtils.getPathWithoutScheme(path))))
+        .thenReturn(true);
+    FileSystem alluxioHadoopFs = new FileSystem(alluxioFs);
+
+    try {
+      alluxioHadoopFs.append(path, 100);
+      fail("append() of existing file is expected to fail");
+    } catch (IOException e) {
+      assertEquals("append() to existing Alluxio path is currently not supported: " + path,
+          e.getMessage());
+    }
+    alluxioHadoopFs.close();
+  }
+
+  @Test
+  public void createWithoutOverwrite() throws Exception {
+    Path path = new Path("/file");
+    alluxio.client.file.FileSystem alluxioFs =
+        mock(alluxio.client.file.FileSystem.class);
+    when(alluxioFs.exists(new AlluxioURI(HadoopUtils.getPathWithoutScheme(path))))
+        .thenReturn(true);
+    when(alluxioFs.createFile(eq(new AlluxioURI(HadoopUtils.getPathWithoutScheme(path))), any()))
+        .thenThrow(new FileAlreadyExistsException(path.toString()));
+    FileSystem alluxioHadoopFs = new FileSystem(alluxioFs);
```

Comment:
I will make this `try-with-resource` to make sure it is closed

Reply:
same comments apply to other parts

Reply:
Done